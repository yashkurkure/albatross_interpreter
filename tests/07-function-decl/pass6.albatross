var eZ7NoaPYlkkLuo5E int := 55393-32066&&27732&&23411==45209|38071^49578|!10454;
var k2O6nc0ieKRNBRNoS string := "uQbISNu9rg4ED";
var tNjn19lagnz int := eZ7NoaPYlkkLuo5E&&10240<>5942>48736+eZ7NoaPYlkkLuo5E<>eZ7NoaPYlkkLuo5E|eZ7NoaPYlkkLuo5E<eZ7NoaPYlkkLuo5E^63982&&5534&46668%10851<>eZ7NoaPYlkkLuo5E||eZ7NoaPYlkkLuo5E||!2191;
var FT37W int := 30740||20689+tNjn19lagnz%38383>=47365==tNjn19lagnz>=46741-eZ7NoaPYlkkLuo5E-tNjn19lagnz<=4495>63977>=42547+37956<tNjn19lagnz&65270;
var fkCyIw3pDGcNocSrGGUs int := 24347&39493>eZ7NoaPYlkkLuo5E-FT37W^!19858>16198>=eZ7NoaPYlkkLuo5E==eZ7NoaPYlkkLuo5E&50151&12849==FT37W==37946*4930&&FT37W||eZ7NoaPYlkkLuo5E;
fun o1yLsYP5xaw int () {return 28472==tNjn19lagnz%55959==tNjn19lagnz&&64575>eZ7NoaPYlkkLuo5E&&4968|36721||fkCyIw3pDGcNocSrGGUs%fkCyIw3pDGcNocSrGGUs<>tNjn19lagnz-37965<14885;}
fun mG0 int () {return !!14696&&FT37W-tNjn19lagnz&&60553^44524^fkCyIw3pDGcNocSrGGUs<>3790;}
fun ShXTcLKykR void () {repeat (tNjn19lagnz<=fkCyIw3pDGcNocSrGGUs*FT37W>=12618<=FT37W&&fkCyIw3pDGcNocSrGGUs-fkCyIw3pDGcNocSrGGUs==fkCyIw3pDGcNocSrGGUs&5386<!12912==59759) {if (FT37W>=5093*64071<=8616<=62508&&63131<tNjn19lagnz<=eZ7NoaPYlkkLuo5E<>fkCyIw3pDGcNocSrGGUs*tNjn19lagnz-fkCyIw3pDGcNocSrGGUs*10546|58764) {return;}}}
fun em6ape6j int () {repeat (48649>17870*48309>=16506&59591<>FT37W<>FT37W<=tNjn19lagnz*fkCyIw3pDGcNocSrGGUs<=10986^44297||tNjn19lagnz>=35467|fkCyIw3pDGcNocSrGGUs&fkCyIw3pDGcNocSrGGUs^27638) {return tNjn19lagnz%50893>=eZ7NoaPYlkkLuo5E<>eZ7NoaPYlkkLuo5E>=!eZ7NoaPYlkkLuo5E&&34597%tNjn19lagnz==14928||48923|24906&&tNjn19lagnz|63518<>22651|35212==10213;}}
fun jM77YzZPUEKVg void () {while (!26548*fkCyIw3pDGcNocSrGGUs%tNjn19lagnz<46078%eZ7NoaPYlkkLuo5E^38124|FT37W>tNjn19lagnz) {repeat (tNjn19lagnz<>8056%49952-fkCyIw3pDGcNocSrGGUs-38756^eZ7NoaPYlkkLuo5E<=FT37W-tNjn19lagnz>=49409==21368<=FT37W&&18464>tNjn19lagnz&44085&&56543>eZ7NoaPYlkkLuo5E) {return;}}}
k2O6nc0ieKRNBRNoS:="AG9S2wfPDRS28gVUa";
k2O6nc0ieKRNBRNoS:=k2O6nc0ieKRNBRNoS;
k2O6nc0ieKRNBRNoS:=k2O6nc0ieKRNBRNoS;
k2O6nc0ieKRNBRNoS:="Oo";
return 41566^FT37W%tNjn19lagnz||FT37W|18545&&61748^22323%3214>=24160;
