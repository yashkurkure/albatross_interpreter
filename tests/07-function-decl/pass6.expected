Variable declared "eZ7NoaPYlkkLuo5E" type int
Variable declared "k2O6nc0ieKRNBRNoS" type string
Variable declared "tNjn19lagnz" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable declared "FT37W" type int
Variable read "tNjn19lagnz" type int
Variable read "tNjn19lagnz" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable read "tNjn19lagnz" type int
Variable read "tNjn19lagnz" type int
Variable declared "fkCyIw3pDGcNocSrGGUs" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable read "FT37W" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable read "FT37W" type int
Variable read "FT37W" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Function declared "o1yLsYP5xaw" returns int
Variable read "tNjn19lagnz" type int
Variable read "tNjn19lagnz" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable read "fkCyIw3pDGcNocSrGGUs" type int
Variable read "fkCyIw3pDGcNocSrGGUs" type int
Variable read "tNjn19lagnz" type int
Function declared "mG0" returns int
Variable read "FT37W" type int
Variable read "tNjn19lagnz" type int
Variable read "fkCyIw3pDGcNocSrGGUs" type int
Function declared "ShXTcLKykR" returns void
Variable read "tNjn19lagnz" type int
Variable read "fkCyIw3pDGcNocSrGGUs" type int
Variable read "FT37W" type int
Variable read "FT37W" type int
Variable read "fkCyIw3pDGcNocSrGGUs" type int
Variable read "fkCyIw3pDGcNocSrGGUs" type int
Variable read "fkCyIw3pDGcNocSrGGUs" type int
Variable read "FT37W" type int
Variable read "tNjn19lagnz" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable read "fkCyIw3pDGcNocSrGGUs" type int
Variable read "tNjn19lagnz" type int
Variable read "fkCyIw3pDGcNocSrGGUs" type int
Function declared "em6ape6j" returns int
Variable read "FT37W" type int
Variable read "FT37W" type int
Variable read "tNjn19lagnz" type int
Variable read "fkCyIw3pDGcNocSrGGUs" type int
Variable read "tNjn19lagnz" type int
Variable read "fkCyIw3pDGcNocSrGGUs" type int
Variable read "fkCyIw3pDGcNocSrGGUs" type int
Variable read "tNjn19lagnz" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable read "tNjn19lagnz" type int
Variable read "tNjn19lagnz" type int
Function declared "jM77YzZPUEKVg" returns void
Variable read "fkCyIw3pDGcNocSrGGUs" type int
Variable read "tNjn19lagnz" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable read "FT37W" type int
Variable read "tNjn19lagnz" type int
Variable read "tNjn19lagnz" type int
Variable read "fkCyIw3pDGcNocSrGGUs" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable read "FT37W" type int
Variable read "tNjn19lagnz" type int
Variable read "FT37W" type int
Variable read "tNjn19lagnz" type int
Variable read "eZ7NoaPYlkkLuo5E" type int
Variable written "k2O6nc0ieKRNBRNoS" type string
Variable read "k2O6nc0ieKRNBRNoS" type string
Variable written "k2O6nc0ieKRNBRNoS" type string
Variable read "k2O6nc0ieKRNBRNoS" type string
Variable written "k2O6nc0ieKRNBRNoS" type string
Variable written "k2O6nc0ieKRNBRNoS" type string
Variable read "FT37W" type int
Variable read "tNjn19lagnz" type int
Variable read "FT37W" type int
